src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -30,6 +31,7 @@ def get_attrs(self, item_list, user, **kwargs):
 
     def serialize(self, obj, attrs, user):
         env = obj.snuba_query.environment
+        aggregation = aggregate_to_query_aggregation.get(obj.snuba_query.aggregate).value
```

Comment:
I think this lookup will fail because with the new performance selector we will be saving aggregates we don't recognize.

Reply:
This will fail eventually, but currently there's no way to pass those string aggregates to the backend, so is fine for now. I'll tackle this once I start those api changes.